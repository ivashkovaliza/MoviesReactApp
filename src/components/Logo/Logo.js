import React from "react";
import './Logo.scss';

const Logo = () => {
  return <span className={'logo'}><span className={'logo--bold'}>netflix</span>roulette</span>
};

export default Logo;
