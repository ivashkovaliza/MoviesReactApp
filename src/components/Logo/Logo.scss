.logo {
  font-size: 22px;
  font-weight: 200;
  color: #f65261;

  &--bold {
    font-weight: bold;
  }
}
