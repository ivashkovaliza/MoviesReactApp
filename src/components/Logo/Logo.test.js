import React from 'react';
import Logo from './Logo';

describe('Logo', () => {
  test('renders correctly', () => {
    const component = shallow(
      <Logo />
    );
    expect(component).toMatchSnapshot();
  });
});
