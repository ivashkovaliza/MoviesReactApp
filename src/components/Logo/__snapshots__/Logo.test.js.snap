// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders correctly 1`] = `
<span
  className="logo"
>
  <span
    className="logo--bold"
  >
    netflix
  </span>
  roulette
</span>
`;
